website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md

```diff
@@ -0,0 +1,74 @@
+---
+title: What’s new with authentik - March 2023
+slug: 2023-03-23-whats-new-with-authentik-march-2023
+authors:
+    - name: Jens Langhammer
+      title: CTO at Authentik Security Inc
+      url: https://github.com/BeryJu
+      image_url: https://github.com/BeryJu.png
+    - name: Tana Berry
+      title: Sr. Technical Content Editor at Authentik Security Inc
+      url: https://github.com/tanberry
+      image_url: https://github.com/tanberry.png
+tags:
+    - announcement
+hide_table_of_contents: false
+---
+
+In a blog from last November 2022, titled “[Next steps for authentik](https://goauthentik.io/blog/2022-11-02-the-next-step-for-authentik)”, I wrote about the launch of [Authentik Security](https://goauthentik.io/), our open core company built around the open source project [authentik](https://github.com/goauthentik/authentik).
+
+In this post, we’d like to provide updates on our progress in building out Authentik Security the company, ramping up the feature set in our open source identity provider, and taking the first steps in developing/offering an enterprise-level feature set for the Cloud or self-hosting. We are enthusiastic about our path forward and our plans to take authentik from a project to a product.
+
+<!--truncate-->
+
+## Company Updates
+
+Authentik Security is now officially almost 6 months old. The energy and vitality of our community-based authentik project continues, and now with a growing staff we can keep the open source product expanding with new releases and also work on launching and growing the company. Mixed into that balance is defining our feature sets for each plan, and developing new functionality for companies that want to deploy authentik at the enterprise level, either self-hosted or on the Cloud as a Saas offering by Authentik Security.
+
+We have a lot of exciting work in front of us. Check out our [job posts](https://goauthentik.io/jobs/) and consider joining us!
+
+## New features for authentik
+
+Authentik Security aims to always be open and transparent, and our trust in our community’s awesomeness means that we realize you all are experts in the field, the ones working in the security and identity management industry every day (and night!), so we look forward to strengthening our collaboration and communication with you, our user.
+
+We have a roadmap with several new features, and we want to hear your opinions on them. You can write us directly at hello@goauthentik.io or open a GitHub issue. For more targeted conversations, we will reach out to schedule calls with users who want to provide more in-depth collaboration and feedback.
```

Comment:
@BeryJu  do you think we should use a mail:to link for the email address?

Reply:
Docusaurus seems to do this automatically